### ISSUE TITLE
Different results between tfsharp and tensorflow

### ISSUE THREAD

**Issue by guoguoguilai** [migueldeicaza/TensorFlowSharp#456]
I trained a simple network with tensorflow==1.14, and it worked pretty well. The main test code are as follows:

**graph = tf.get_default_graph()
pred = graph.get_tensor_by_name("dense/Softmax:0")
txt = "train/heng/35.txt"
data = process_a_txt(txt)
res = sess.run(pred, feed_dict={'flatten_input:0': data.reshape([1,16, 16])})#test_images
print(res)**

and the result is :**[[0.0168 0.0042 0.9476 0.0274 0.004 ]]**
However ,when I loaded model with tfsharp, the result was different. This is the code:

**TFShape shape = new TFShape(1, size , size);
var tensor = TFTensor.FromBuffer(shape, contents, 0, size * size);
using (var sess = new TFSession(graph))
{
      var runner = sess.GetRunner();
      runner.AddInput(graph["flatten_input"][0], tensor);
      var r = runner.Run(graph.Softmax(graph["dense/Softmax"][0]));
      var v = (float[,])r.GetValue();
      for (int i = 0; i < v.Length; i++)
      {
           Debug.Log(v[0, i]);
      }
}**
And it got: **[0.1522539,0.150988,0.3921793,0.1536978,0.1508811]**
Although the class was correct, the probability are not the same. And I am pretty sure that the input are the same.

**Comment by guoguoguilai** [comment]
It seems that Compared to tf, an softmax layer was added in tfsharp.

**Comment by guoguoguilai** [comment]
It was my mistake. Solved.
